pom.xml

```diff
@@ -51,6 +51,12 @@
   </organization>
 
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
+    <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>32.1.2-jre</version>
+    </dependency>
```

Comment:
Please do not update unrelated dependencies with new features.

Reply:
Yeah, this update is only for Java 17 support